private-templates-service/client/src/components/Dashboard/Dashboard.tsx

```diff
@@ -22,26 +27,31 @@ const mapDispatchToProps = (dispatch: any) => {
   return {
     setPage: (currentPageTitle: string) => {
       dispatch(setPage(currentPageTitle));
-    }
-  };
-};
+    },
+    getTemplates: () => { dispatch(getAllTemplates()) }
+  }
+}
 
 interface State {
   isPreviewOpen: boolean;
 }
 
 interface Props {
   isAuthenticated: boolean;
+  user?: UserType;
+  isSearch: boolean;
+  batchTemplates: BatchTemplatesState;
   authButtonMethod: () => Promise<void>;
   setPage: (currentPageTitle: string) => void;
-  isSearch: boolean;
+  getTemplates: () => void;
 }
 
 class Dashboard extends React.Component<Props, State> {
   constructor(props: Props) {
     super(props);
     this.state = { isPreviewOpen: false };
     props.setPage("Dashboard");
+    props.getTemplates();
```

Comment:
nit: consistency - i've seen 'allTemplates', 'batchTemplates' and now 'templates'. Choose one :)